### PR TITLE
Rewards allocation for Nov 1st 2022

### PR DESCRIPTION
**Pull request by r-czajkowski**
This PR adds a merkle tree computed in https://github.com/threshold-network/merkle-distribution/pull/37 to Threshold Token Dashboard.

### CODE REVIEW

File: src/merkle-drop/rewards.json [hunk 1]
```diff
@@ -1,2588 +1,2769 @@
 {
-  "totalAmount": "220448748542061979600986806",
-  "merkleRoot": "0x779158ff622e544fe3652d586290691b59c567c3c366394740ad5ae8c22ff97b",
+  "totalAmount": "256480309535848311798474676",
+  "merkleRoot": "0x4ce540139e54d66475fa40f5f4bf5398c17937b54960852cb56734c023112dc3",
```
Comment (michalsmiarowski) [inline src/merkle-drop/rewards.json:3]: I think the values are not correct 🤔  There are differenv values for `totalAmount` and `merkleRoot` in https://github.com/threshold-network/merkle-distribution/pull/37 (`MerkleDIst.json` file)

Your values:
```
"totalAmount": "256480309535848311798474676",
"merkleRoot": "0x4ce540139e54d66475fa40f5f4bf5398c17937b54960852cb56734c023112dc3",
```
  
Values in `merkle-distribution`:
```
"totalAmount": "256710294590339327077459526",
"merkleRoot": "0xf2616aeb5f4ddf8e53afdbc49bb24d628dd4a7bbb6e9d16df39dc9b1a6419fc6",
```

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
I think values here don't match the values in https://github.com/threshold-network/merkle-distribution/pull/37. Let's verify it before the merge. See the comment below.

**Review by michalsmiarowski — APPROVED** [review]
LGTM! 👍

**Review by manumonti — APPROVED** [review]
LGTM ✌️

**Pull request merged**